.github/workflows/integration-test.yml

```diff
@@ -55,7 +66,18 @@ jobs:
     with:
       database: ${{ github.event.inputs.database2 }}
       engine: ${{ github.event.inputs.engine2 }}
-      account: ${{ github.event.inputs.account }}
+      account: ${{ vars. FIREBOLT_ACCOUNT_V1 }}
+    secrets:
+      FIREBOLT_CLIENT_ID_STG_NEW_IDN: ${{ secrets.FIREBOLT_CLIENT_ID_STG_NEW_IDN }}
+      FIREBOLT_CLIENT_SECRET_STG_NEW_IDN: ${{ secrets.FIREBOLT_CLIENT_SECRET_STG_NEW_IDN }}
+
+  run-integration-tests-engine2:
+    if: ${{ github.event.inputs.run-database-with-engine-v2 == 'true' }}
+    uses: ./.github/workflows/integration-test-v2.yml
+    with:
+      database: ${{ github.event.inputs.database-with-engine-v2 }}
+      engine: ${{ github.event.inputs.engine_v2_fb_2_0 }}
+      account: ${{ vars. FIREBOLT_ACCOUNT_V2 }}
```

Comment:
You have a space after `vars. `, let's remove it as I think this will fail.
This had a space in my previous comment so sorry about the confusion there.

Reply:
yes, fixed. I have seen it myself as well. Thank you.